### PR TITLE
Adding AllDifferentIndex and ArratContainNumber program

### PR DESCRIPTION
**Pull request by SnehalWAug19**

### CODE REVIEW

File: src/SnehalW/AllDifferentIndex.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package SnehalW;
+
+public class AllDifferentIndex {
+
+	int array1[] = {10,2,9,14,3};
+	int array2[] = {10,2,18,14,3};
+	int i ,j;
+		void differIndex()
```
Comment (ghost) [inline src/SnehalW/AllDifferentIndex.java]: Hi @SnehalWAug19 
Here why we take two for loop in single for loop work can be done..
think you need to compare same index number in both array... for that you should write one for loop and check for which index number are not matching...you are getting my point ?

Please Correct the code!

File: src/SnehalW/ArratContainNumber.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package SnehalW;
+
+public class ArratContainNumber
+{
+
```
Comment (ghost) [inline src/SnehalW/ArratContainNumber.java:5]: Hi @SnehalWAug19 
Here Logic is wrong...
here you are checking the not condition.... if that number not exist in array you breaking the for loop but... check for the any number which is in array, For that your logic will be wrong...

you should check the if that number exist in array then fine(do not print or break) but if not (else) condition you need to print and break that condition...

Hope your getting my point...

File: src/SnehalW/ArrayEquality.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+{
+	int array1[] = {21, 57, 11, 37, 24};
+	int array2[] = {21, 57, 11, 37, 24};
+	int count;
+	void equality()
+	{
+		for(int i=0;i<array1.length;i++)
+		{
+			for(int j=0;j<array2.length;j++)
+			{
```
Comment (ghost) [inline src/SnehalW/ArrayEquality.java:13]: Hi @SnehalWAug19 
Here good you to see you are comparing the length of both array..
best way to compare both's length before the for loop.. i mean 
if(both's lnegth is same){
     //Add your code
}else{
   SOP("Length is miss match");
}

this is correct just anothere way..

**Review by ghost — COMMENTED** [review]
Hi @SnehalWAug19 
Please work on the comment and raised PR again...
Other Codes are good.. expected logic you used...
keep it up!

try to follow best practice.

These are main..
Class name's first letter should be Capital for Example : class Student
method's name first letter should be small for Example : findSecondMaxFromArray()
variable name first letter should be small for Example : int rollNum

File: src/SnehalW/AllDifferentIndex.java (hunk 1 above)
Comment (SnehalWAug19) [reply src/SnehalW/AllDifferentIndex.java]: Yes harsh, you r correct program is working by taking  for loop also.Updated the program and pushed in git again

File: src/SnehalW/ArrayEquality.java (hunk 3 above)
Comment (SnehalWAug19) [reply src/SnehalW/ArrayEquality.java:13]: thanks harsh

**Pull request merged**